.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: smtp-scm-1.rockspec
+
+  publish-tag:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+    env:
+      CMAKE_LDOC_FIND_REQUIRED: 'YES'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rosik/setup-tarantool@v1
+        with:
+          tarantool-version: '2.8'
+
+      # Setup ldoc
+      - run: tarantoolctl rocks install ldoc
+          --server=https://tarantool.github.io/LDoc/
+      - run: echo $PWD/.rocks/bin >> $GITHUB_PATH
+
+      # Make a release
+      - run: echo TAG=${GITHUB_REF##*/} >> $GITHUB_ENV
+      - run: tarantoolctl rocks new_version --tag ${{ env.TAG }}
+      - run: tarantoolctl rocks install smtp-${{ env.TAG }}-1.rockspec
```

Comment:
Is it to check whether `s/branch = 'master'/tag = '<...>'` breaks anything? Hmm, don't know how it may occur.

Reply:
It should be done to pack binary rock. Removed.